checkov/terraform/checks/resource/gcp/GKEPublicControlPlane.py

```diff
@@ -20,7 +20,7 @@ def scan_resource_conf(self, conf):
         if 'master_authorized_networks_config' in conf.keys():
             if isinstance(conf['master_authorized_networks_config'][0], dict) and 'cidr_blocks' in conf['master_authorized_networks_config'][0]:
                 for cidr_block_conf in conf['master_authorized_networks_config'][0]['cidr_blocks']:
-                    if '0.0.0.0/0' in cidr_block_conf['cidr_block']:
+                    if type(cidr_block_conf) is dict and '0.0.0.0/0' in cidr_block_conf['cidr_block']:
```

Comment:
@rotemavni We use `isinstance()` as a convention - can you refactor please?

Reply:
@nimrodkor done